Python/Product/Cookiecutter/Shared/Infrastructure/OutputWindowRedirector.cs

```diff
@@ -58,7 +58,7 @@ sealed class OutputWindowRedirector : Redirector {
         /// not be found or created.</exception>
         public static OutputWindowRedirector GetGeneral(IServiceProvider provider) {
             if (_generalPane == null) {
-                _generalPane = Get(provider, VSConstants.OutputWindowPaneGuid.GeneralPane_guid, "General");
+                _generalPane = Get(provider, VSConstants.OutputWindowPaneGuid.GeneralPane_guid, Strings.Settings_General_Category);
```

Comment:
I assume these are already defined

Reply:
Already defined and localized. The fix in master adds specific new strings, but we won't have time to get those fully updated for 15.9